tests/modules/test_catalog_csv.py

```diff
@@ -95,7 +95,7 @@ def test_with_replace_no_files():
 
     with raises(
         HttpException,
-        match=r'form-data should contain at least one of "items", "variations", "item_groups'
+        match=r"Something unexpected went wrong here. We're looking into it. Sorry for the inconvenience!"
```

Comment:
Might need to raise this to backend.

Reply:
@Mudaafi Let's raise this with backend.

Reply:
Fix has been deployed to backend so I've reverted the changes.